contrib/platform/src/com/sun/jna/platform/win32/Msi.java

```diff
@@ -218,4 +251,209 @@ public interface Msi extends StdCallLibrary {
      *  ERROR_SUCCESS - A value was enumerated.
      */
     int MsiEnumComponents(WinDef.DWORD iComponentIndex, char[] lpComponentBuf);
+
+    /**
+     * The MsiCloseHandle function closes an open installation handle.
+     *
+     * @param hAny
+     *   Specifies any open installation handle.
+     *
+     * @return
+     *   ERROR_INVALID_HANDLE - An invalid handle was passed to the function.
+     *   ERROR_SUCCESS - The function succeeded.
+     */
+    int MsiCloseHandle(Pointer hAny);
```

Comment:
Where does the definition of `hAny` come from? I see this in `Msi.h`:

```c
typedef unsigned long MSIHANDLE;
```

If there is no other definition, it might be worth defining `MSIHANDLE` as a `NativeMapped` type and create `MSIHandleByReference` in the same as `IntByReference` is defined.

This would need to be updated for all call sites here.